sdk/webpubsub/azure-webpubsub/azure/webpubsub/_azure_web_pub_sub_service_restapi.py

```diff
@@ -0,0 +1,84 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from typing import TYPE_CHECKING
+
+from azure.core import PipelineClient
+from azure.core.protocol import HttpResponse
+from msrest import Deserializer, Serializer
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import,ungrouped-imports
+    from typing import Any, Dict, Optional
+
+    from azure.core.protocol import HttpRequest
+
+from ._configuration import AzureWebPubSubServiceRESTAPIConfiguration
+from .operations import HealthApiOperations
+from .operations import WebPubSubOperations
```

Comment:
should the swagger have operation groups?

side note: if we go the `--low-level-client` route, don't generate operation groups on service client

Reply:
With `--low-level-client`, don't use `--payload-flattening-threshold`